core/src/apps/cardano/README.md

```diff
@@ -312,7 +322,7 @@ When one wants to spend funds from the Plutus script address (which is possible
 You can use a combination of [cardano-node](https://github.com/input-output-hk/cardano-node) and cardano-cli (part of the cardano-node repo) to submit a transaction.
 
 ## Serialization format
-Cardano uses [CBOR](https://www.rfc-editor.org/info/rfc7049) as a serialization format. [Here](https://github.com/input-output-hk/cardano-ledger/blob/9c3b4737b13b30f71529e76c5330f403165e28a6/eras/alonzo/test-suite/cddl-files/alonzo.cddl) is the [CDDL](https://tools.ietf.org/html/rfc8610) specification for after Plutus script support has been added.
+Cardano uses [CBOR](https://www.rfc-editor.org/info/rfc7049) as a serialization format. [Here](https://github.com/input-output-hk/cardano-ledger/blob/1cbf1fc2bb005a8206e5b5a7cdf44d35baaca455/eras/babbage/test-suite/cddl-files/babbage.cddl) is the [CDDL](https://tools.ietf.org/html/rfc8610) specification for Babbage era.
```

Comment:
Good idea (especially because of the tagged embedded CBOR), I added a tx example with Plutus elements in https://github.com/vacuumlabs/trezor-firmware/pull/78/commits/64eed36fc454b1098e16de8d055c92b387225abd.

> perhaps we can remove even the existing description and just mention that one of the CBOR tools can be used to view details

I prefer keeping the structure description that we have - it is richer than what CBOR tools can display (it contains the tx element names).